spring-batch-docs/asciidoc/whatsnew.adoc

```diff
@@ -4,323 +4,28 @@
 
 [[whatsNew]]
 
-== What's New in Spring Batch 4.1
+== What's New in Spring Batch 4.2
 
-The Spring Batch 4.1 release adds the following features:
+Spring Batch 4.2 adds the following features:
 
-* A new `@SpringBatchTest` annotation to simplify testing batch components
-* A new `@EnableBatchIntegration` annotation to simplify remote chunking and partitioning configuration
-* A new `JsonItemReader` and `JsonFileItemWriter` to support the JSON format
-* Add support for validating items with the Bean Validation API
-* Add support for JSR-305 annotations
-* Enhancements to the `FlatFileItemWriterBuilder` API
+* Support for batch metrics with https://micrometer.io[Micrometer]
+* Improved documentation
 
-[[whatsNewTesting]]
-=== `@SpringBatchTest` Annotation
+[[whatsNewMetrics]]
+=== Batch metrics with Micrometer
 
-Spring Batch provides some nice utility classes (such as the `JobLauncherTestUtils` and
-`JobRepositoryTestUtils`) and test execution listeners (`StepScopeTestExecutionListener`
-and `JobScopeTestExecutionListener`) to test batch components. However, in order
-to use these utilities, you must configure them explicitly. This release introduces
-a new annotation named `@SpringBatchTest` that automatically adds utility beans and
-listeners to the test context and makes them  available for autowiring,
-as the following example shows:
+This release introduces a new feature that allows you to monitor your batch jobs
+using Micrometer. By default, Spring Batch will collect metrics like job duration,
+step duration, item read/write throughput, etc and register them in Micrometer's
```

Comment:
Please replace with "step duration, item read and write throughput, and others) and register them in Micrometer's"